launchable/utils/fail_fast_mode.py

```diff
@@ -0,0 +1,101 @@
+import sys
+from typing import List, Optional, Sequence, Tuple
+
+import click
+
+from .commands import Command
+
+_fail_fast_mode_cache: Optional[bool] = None
+
+
+def set_fail_fast_mode(enabled: bool):
+    global _fail_fast_mode_cache
+    _fail_fast_mode_cache = enabled
+
+
+def is_fail_fast_mode() -> bool:
+    if _fail_fast_mode_cache:
+        return _fail_fast_mode_cache
+
+    # Default to False if not set
+    return False
+
+
+def warning_and_exit_if_fail_fast_mode(message: str):
```

Comment:
nit: warning is a noun, so I guess the verb, warn is better?

```suggestion
def warn_and_exit_if_fail_fast_mode(message: str):
```

Reply:
https://github.com/launchableinc/cli/pull/1046/commits/016dec9d56105810cea7a0c4bd937ae8b613c82a